### PR TITLE
[debug] add log to show the IP Address when connecting the socket

### PR DESCRIPTION
**Pull request by jayahariv**

### CODE REVIEW

File: Objective-C/Internal/Replicator/CBLWebSocket.mm [hunk 1]
```diff
@@ -440,6 +445,9 @@ - (void) connectToHostWithName: (NSString*)hostname
             return; // Already disconnected
         }
         
+        struct sockaddr_in *addr = (struct sockaddr_in *)_addr->ai_addr;
+        char *ip = inet_ntoa((struct in_addr)addr->sin_addr);
```
Comment (pasin) [inline Objective-C/Internal/Replicator/CBLWebSocket.mm:449]: I think this function is only for IPV4.

Comment (pasin) [reply Objective-C/Internal/Replicator/CBLWebSocket.mm:449]: inet_ntop supports both.

**Review by pasin — CHANGES_REQUESTED** [review]

**Comment by jayahariv** [comment]
Not tested ipv6 address, but ipv4 works fine. will do test today

File: Objective-C/Internal/Replicator/CBLWebSocket.mm [hunk 2]
```diff
@@ -440,6 +445,17 @@ - (void) connectToHostWithName: (NSString*)hostname
             return; // Already disconnected
         }
         
+        struct sockaddr_in *addr = (struct sockaddr_in *)_addr->ai_addr;
+        char addrBuf[MAX(INET_ADDRSTRLEN, INET6_ADDRSTRLEN)];
+        if(addr->sin_family == AF_INET) {
+            if (inet_ntop(AF_INET, &addr->sin_addr, addrBuf, INET_ADDRSTRLEN) != NULL) {
```
Comment (pasin) [inline Objective-C/Internal/Replicator/CBLWebSocket.mm:451]: These two line doesn't match :
char addrBuf[MAX(**INET_ADDRSTRLEN, INET6_ADDRSTRLEN**)];
if (inet_ntop(AF_INET, &addr->sin_addr, addrBuf, **INET_ADDRSTRLEN**)

File: Objective-C/Internal/Replicator/CBLWebSocket.mm [hunk 3]
```diff
@@ -965,6 +987,7 @@ - (void) doRead {
 }
 
 - (void)stream: (NSStream*)stream handleEvent: (NSStreamEvent)eventCode {
+    CBLLogVerbose(WebSocket, @"event code: %lx - stream: %@", (unsigned long)eventCode, stream);
```
Comment (pasin) [inline Objective-C/Internal/Replicator/CBLWebSocket.mm:990]: We have logged on every single event. This is not necessary.

**Review by pasin — CHANGES_REQUESTED** [review]

**Review by pasin — APPROVED** [review]

**Pull request merged**